### PR TITLE
Add compliance check for windows to ensure cross platform build

### PR DESCRIPTION
**Pull request by techknowlogick**
as title

Note: This will fail until windows issues are resolved.

### CODE REVIEW

**Review by zeripath — APPROVED** [review]

**Review by jolheiser — APPROVED** [review]
LGTM after the underlying issue is fixed 😄

**Comment by 6543** [comment]
blocked by https://github.com/go-gitea/gitea/pull/14263

**Comment by techknowlogick** [comment]
🚀

**Pull request merged**
